docs/earth-config/earth-config.md

```diff
@@ -0,0 +1,34 @@
+# Earthly config file
+
+Global configuration values for earth can be stored on disk in the configuration file.
+
+By default, earth reads the configuration file `~/.earthly/config.yaml`; however, it can also be
+overridden with the `--config` command flag option.
+
+Currently only https authentication configuration is supported by the config file.
```

Comment:
Perhaps this should be clarified that only https auth is *configured* here, however both SSH and user-pass auth are supported. We should also point to the auth guide for more info.